core/player/src/controllers/validation/binding-tracker.ts

```diff
@@ -214,46 +205,66 @@ export class ValidationBindingTrackerViewPlugin
       };
     });
 
-    resolver.hooks.afterNodeUpdate.tap(CONTEXT, (node, parent, update) => {
-      if (parent) {
-        addToTree(node, parent);
-      }
+    resolver.hooks.afterNodeUpdate.tap(
+      CONTEXT,
```

Comment:
Minor but we usually reserve all capital variable names for constants

Reply:
it is a constant, already defined further up -> https://github.com/player-ui/player/blob/main/core/player/src/controllers/validation/binding-tracker.ts#L11

Reply:
Ah, my bad. I forgot the first argument of a tap was the tap id. In my opinion we still might want to change that to be something more reflective of it being an ID and not context that comes in with the hook but thats definitely out of scope of this.